source/extensions/filters/http/gzip/gzip_filter.cc

```diff
@@ -191,15 +217,23 @@ bool GzipFilter::isContentTypeAllowed(Http::HeaderMap& headers) const {
 }
 
 bool GzipFilter::isEtagAllowed(Http::HeaderMap& headers) const {
-  return !(config_->disableOnEtagHeader() && headers.Etag());
+  bool isEtagAllowed = !(config_->disableOnEtagHeader() && headers.Etag());
```

Comment:
nit: all variables snake_case. There are other examples in the PR, please audit. Also, this var can be `const bool`. Also other similar examples for this, please audit.

Reply:
Ah yeah. Copy paste fail. Will fix.